dnsext-full-resolver/DNS/Cache/Iterative.hs

```diff
@@ -1101,11 +1097,10 @@ delegationWithCache zoneDom dnskeys dom msg = do
             cacheDS
             cacheNS
             cacheAdds
-            clogLn Log.DEMO verifyColor $ verifyMsg ++ ": " ++ domTraceMsg
             clogLn Log.DEMO Nothing $ ppDelegation (delegationNS x)
             return x
 
-    logLn Log.INFO $ "delegationWithCache: " ++ domTraceMsg ++ ", " ++ verifyMsg
+    logLn Log.DEMO $ "delegationWithCache: " ++ domTraceMsg ++ ", " ++ verifyMsg
```

Comment:
If you don't mind, how about adding color again to the validation results?

https://github.com/kazu-yamamoto/dnsext/commit/be09797e44a1b672423df07d193f0cbc5ae43050

Reply:
Nice catch and done!